com.avaloq.tools.ddk.sample.helloworld/dsl-gen/com/avaloq/tools/ddk/sample/helloworld/validation/ExecutionEnvironmentStandaloneSetup.java

```diff
@@ -18,7 +18,7 @@ public class ExecutionEnvironmentStandaloneSetup implements ICheckValidatorStand
   private static final String GRAMMAR_NAME = "com.avaloq.tools.ddk.sample.helloworld.HelloWorld";
   private static final String CATALOG_FILE_PATH = "com/avaloq/tools/ddk/sample/helloworld/validation/ExecutionEnvironment.check";
 
-  @Override
+  /** {@inheritDoc} */

```

Comment:
The annotation /** {@inheritDoc} */ is not correct. If it is an override, there is nothing to inherit for the documentation. Could you remove that from this place and the rest in this PR?

Reply:
Ok.